hack/make/build-deb

```diff
@@ -86,6 +86,16 @@ set -e
 		cat >> "$DEST/$version/Dockerfile.build" <<-EOF
 			RUN cp -aL hack/make/.build-deb debian
 			RUN { echo '$debSource (${debVersion}-0~${suite}) $suite; urgency=low'; echo; echo '  * Version: $VERSION'; echo; echo " -- $debMaintainer  $debDate"; } > debian/changelog && cat >&2 debian/changelog
+		EOF
+		# Remove the following case-based substitution when none of these are supported, and the TasksMax value is uncommented in docker.service
+		case "$version" in
+		debian-jessie|debian-wheezy|ubuntu-precise|ubuntu-trusty|ubuntu-wily)
+			;;
+		*)
+			echo "RUN sed -i -- 's/#TasksMax=infinity/TasksMax=infinity/' contrib/init/systemd/docker.service" >> "$DEST/$version/Dockerfile.build"
```

Comment:
Just found out that this leads to an armhf deb package for raspbian-jessie with binaries marked as "unsupported". The armhf debian-jessie is fine without that "unsupported" marker.
I'll make a PR for line 92 to make #21895 work again.